Packs/NonSupported/Integrations/integration-VirusTotal.yml

```diff
@@ -245,8 +245,8 @@ script:
         var o = res.obj;
         var ec = {};
         if (o.response_code === 0) {
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"] = {Indicator: hash, Type: 'hash', Vendor: 'VirusTotal', Score: 0};
-            ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"] = {Indicator: hash, Type: 'file', Vendor: 'VirusTotal', Score: 0};
+            ec["DBotScore"] = [{Indicator: hash, Type: 'hash', Vendor: 'VirusTotal', Score: 0},
```

Comment:
remove `hash`

Reply:
The fix in this file is ok. No need to remove hash.

Reply:
done